cmd/buildkitd/config/config.go

```diff
@@ -38,6 +38,14 @@ type Config struct {
 		Dockerfile DockerfileFrontendConfig `toml:"dockerfile.v0"`
 		Gateway    GatewayFrontendConfig    `toml:"gateway.v0"`
 	} `toml:"frontend"`
+
+	System *SystemConfig `toml:"system"`
+}
+
+type SystemConfig struct {
+	// PlatformCacheMaxAge controls how often supported platforms
+	// are refreshed by rescanning the system.
+	PlatformsCacheMaxAge *Duration `toml:"platformsCacheMaxAge"`
```

Comment:
Is it possible to always use cache by setting `platformsCacheMaxAge = -1` with this custom type?

Reply:
Yes, I tested this